pages/clustering/replication.mdx

```diff
@@ -421,7 +421,7 @@ The result has information regarding each individual replica:
 1. replica's name
 2. address
 3. type (sync/async)
-4. system information (available only when using experimental replication feature)
+4. system information (available only when experimental replication is enabled)
```

Comment:
If it's not called experimental replication anymore, then it shouldn't be experimental replication here as well.

Reply:
This is system replication which is currently experimental :D